ci/install-3.5_NUMPY_DEV.sh

```diff
@@ -12,7 +12,8 @@ pip uninstall numpy -y
 # these wheels don't play nice with the conda libgfortran / openblas
 # time conda install -n pandas libgfortran openblas || exit 1
 
-time sudo apt-get $APT_ARGS install libatlas-base-dev gfortran
+# Not going to work in containers
```

Comment:
when you get this to work, you can just delete these lines. a comment is fine too, but don't want people uncommenting it unless they really understand what is going on (so put the comment in .travis.yml where it will be paid a bit more attention).